recipes/flash-spi.md

```diff
@@ -55,3 +55,37 @@ Make sure you remove the microSD card containing the `u-boot-flash-spi` image, o
 5. PXE
 
 If you're currently running the OS from microSD, and want to switch to a USB/SSD drive, follow the [instructions on this page](https://forum.pine64.org/showthread.php?tid=4971).
+
+### 4. Fix for SPI Flashing failures
+
+If, for any reason, your SPI flashing gets interrupted during its process, you may experience a "frozen" device.
+
+To solve this, on the Rock64 follow these instructions:
+
+1. Go to [this](http://wiki.pine64.org/index.php/NOOB#Step-by-Step_Instructions_to_Flashing_MicroSD_Cards) guide to create a new ayufan bootable Linux SD card
+    Or, in short, download the latest ayufan Linux distribution from [here](https://github.com/ayufan-rock64/linux-build/releases/tag/0.6.44), and use [etcher](https://etcher.io/) to flash you SD card
```

Comment:
Any reason to not use `https://github.com/ayufan-rock64/linux-build/releases/latest` as that will redirect to the release that is marked 'release' (as opposed to pre-release)?